CONTRIBUTING.md

```diff
@@ -89,3 +89,83 @@ Reference [pre-commit's installation instructions](https://pre-commit.com/#insta
 ```
 
 See `.pre-commit-config.yaml` for a list of configured linters and fixers.
+
+## Development Environment
+
+### Running Docker Containers
+
+When `docker-compose up` is run from the `deploy/codex/` directory, several Docker containers are started. These are the connected components of the Codex application.
+
+You can view them by using the command `docker ps -a` and see something like this: 
+
+```
+CONTAINER ID   IMAGE                          COMMAND                  CREATED             STATUS                       PORTS                                 NAMES
+e3939c800021   codex_houston                  "/docker-entrypoint.…"   About an hour ago   Up About an hour             0.0.0.0:83->5000/tcp                  codex_houston_1
+5c0b2ab64bdc   dpage/pgadmin4                 "/entrypoint.sh"         22 hours ago        Up About an hour             443/tcp, 0.0.0.0:8000->80/tcp         codex_pgadmin_1
+4761da5d9168   redis:latest                   "docker-entrypoint.s…"   22 hours ago        Up About an hour             6379/tcp                              codex_redis_1
+9560f26f0227   wildme/edm:latest              "/usr/local/tomcat/b…"   22 hours ago        Up About an hour             0.0.0.0:81->8080/tcp                  codex_edm_1
+566e59074f42   nginx:latest                   "/docker-entrypoint.…"   22 hours ago        Up About an hour             0.0.0.0:84->80/tcp                    codex_www_1
+7925cbde4337   wildme/wildbook-ia:nightly     "/docker-entrypoint.…"   22 hours ago        Up About an hour             0.0.0.0:82->5000/tcp                  codex_wbia_1
+77624c19e02d   postgres:10                    "docker-entrypoint.s…"   22 hours ago        Up About an hour             5432/tcp                              codex_db_1
+ba8962d4cf8e   gitlab/gitlab-ce:13.9.3-ce.0   "/assets/wrapper"        22 hours ago        Up About an hour (healthy)   22/tcp, 443/tcp, 0.0.0.0:85->80/tcp   codex_gitlab_1
+f259ede38b22   node:latest                    "/docker-entrypoint.…"   22 hours ago        Up About an hour                                                   codex_dev-frontend_1
+```
+
+These containers are available to enter on the command line using `docker exec -it [CONTAINER NAME] \bin\bash`. This command will grant you command line access as a root user for 
```

Comment:
Surely it should be /bin/bash not \bin\bash

Reply:
There's a difference between using `docker exec` and `docker-compose exec`, with `docker exec`, you need `-it` in order to get an interactive shell (`--interactive --tty`).  Also as Howard said, it should be `/bin/bash`.  I think it's easier to use `docker-compose exec`.  With `docker exec`, you need to use something like `codex_houston_1` but with `docker-compose exec`, you can just use `houston` as the name.